pdf_rendering.py

```diff
@@ -1,239 +1,378 @@
 import fpdf
 import copy
+#TODO OVERALL: MODIFY TEXT OUTPUT AS MORE INFORMATION BECOMES AVAILABLE IN INVOICE
```

Comment:
Lower case should be enough to read them but the description is quite generic, please be more specific.